src/Integration/SLCore/SLCoreConstantsProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.ComponentModel.Composition;
+using SonarLint.VisualStudio.Integration;
+using SonarLint.VisualStudio.SLCore.Service.Lifecycle.Models;
+
+namespace SonarLint.VisualStudio.SLCore.Configuration
+{
+    [Export(typeof(ISLCoreConstantsProvider))]
+    [PartCreationPolicy(CreationPolicy.Shared)]
+    public class SLCoreConstantsProvider : ISLCoreConstantsProvider
+    {
+        public ClientConstantsDto ClientConstants => new ClientConstantsDto("SonarLint for Visual Studio", $"Visual Studio - {VisualStudioHelpers.VisualStudioVersion}");
+
+        public FeatureFlagsDto FeatureFlags => new FeatureFlagsDto(true, true, false, true, false, false, true);
+
+        //We do not support telemetry now
```

Comment:
Set something that would explicitly indicate that this telemetry is not to be sent (so that if we accidentally send it, we can easily filter it out)